CONTRIBUTING.md

```diff
@@ -1,113 +1,111 @@
 
-[English]: https://groups.google.com/a/nervos.org/d/forum/ckb-dev
-[Chinese]: https://groups.google.com/a/nervos.org/d/forum/ckb-dev-zh
 
-Please note we have a code of conduct, please follow it in all your interactions with the project.
 
-### Report Issue
+Did you have problems implementing CKB or do you have a feature request? We would love to hear about it! Here are the most efficient ways to report an issue:
 
-* Read [known issues](https://github.com/nervosnetwork/ckb/projects/2) to see whether the issue is already addressed there.
+# Report Issue
 
-* **Do not open up a GitHub issue to report security vulnerabilities**. Instead,
-  refer to the [security policy](SECURITY.md).
+- **Determine where to submit**
 
-* Search existing issues to see whether others had already posted a similar issue.
+CKB projects are spread across multiple repositories. Please file your issue within the correct repository. If you're not sure which repository is the correct one, check the [list of related projects](https://www.notion.so/cryptape/Resources-10217c86ec47455d9220567bd75d180c).
 
-* When creating a new issue, be sure to include a title and clear description. It is appreciated that if you can also attach as much relevant information as possible, such as version, environment, reproducing steps, samples.
+- **Find existing issues**
 
-* If possible, use the relevant bug report templates to create the issue.
+Before reporting a new issue, please search the [open issues](https://github.com/nervosnetwork/ckb/projects/2) to see if someone has already submitted the issue or feature. If you find a similar issue, please comment on it and add your response. Use the following reactions instead of "+1" comments:
 
-### Send PR
+- 👍 - vote
+- 👎 - veto
+
+If you can't find an existing issue describing your bug or functionality, please create a new issue using the guidelines below.
+
+### Well-Written Bug Reports and Feature Requests
+
+Submit one issue per bug or feature request. Don't list multiple bugs or feature requests in the same issue. Don't add your question as a comment to an existing question unless it's for the same input. Many problems look similar but have different causes. The more information you provide, the more likely you are to successfully reproduce the problem and find a solution.
+
+Please include the following in your submission:
 
-* See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+1. **Do not open up a GitHub issue to report security vulnerabilities**. Instead,
+refer to the [security policy](https://github.com/nervosnetwork/ckb/blob/develop/SECURITY.md)
+2. When creating a new issue, be sure to include **a title and clear description**. It is appreciated that if you can also attach as much relevant information as possible, including:
+- Version
```

Comment:
we don't need to list this kind of information here, because the issue template already has these options?

Excessively long documents can also intimidate new contributors.